### DISCUSSION TITLE
How to add new pages to the navigation?

### DISCUSSION THREAD

**Discussion by ty505** [datenstrom/community#602] [Ask a question] [answered]
How can this be done?

**Comment by IngoBelka** [comment, marked as answer]
https://datenstrom.se/yellow/help/adjusting-content

Simply add a new subfolder in the content folder and store a page.md in it. See naming convention in the link above.

**Reply by annaesvensson** [reply]
That link has moved to https://datenstrom.se/yellow/help/how-to-change-the-content
